src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -64,14 +69,15 @@ object DepositHandler {
       git      <- initGit(tempDir)
       _        <- extractBag(mimeType)
       bagitDir <- getBagDir(tempDir)
+      _        <- resolveFetchItems(bagitDir)
       _        <- checkBagValidity(bagitDir)
       _        <- DepositProperties.set(id, "SUBMITTED", "Deposit is valid and ready for post-submission processing", lookInTempFirst = true)
       _        <- commitSubmitted(git, tempDir)
       dataDir  <- moveBagToStorage()
     } yield ())
     .recover {
       case InvalidDepositException(_, msg, cause) =>
-        log.warn(s"[$id] Invalid deposit", cause)
+        log.error(s"[$id] Invalid deposit", cause)
```

Comment:
OK, could also be `error`. I am not sure about the best criterion here so, fine, change it.